unordered-containers.cabal

```diff
@@ -11,7 +11,7 @@ description:
 license:        BSD3
 license-file:   LICENSE
 author:         Johan Tibell
-maintainer:     johan.tibell@gmail.com
+maintainer:     johan.tibell@gmail.com, David.Feuer@gmail.com
```

Comment:
@emilypi and I also help with maintenance now, but @treeowl is the "primary maintainer" IMHO, so this addition seems right to me.

@tibbe Are you still an active maintainer, too?

Reply:
Not anymore but I'm happy to answer questions about the design decisions involved in the library.

Reply:
Many thanks for being so responsive with the questions @tibbe! :)

I was briefly considering whether @emilypi and I should also be added, but I think that can wait until things have settled a bit. Do you agree @emilypi?

I also came up with "David Feuer and the unordered containers" which I think would be a great band name! ;)

Reply:
I'm good with being added now or later. Maybe after a few legit PR's between us 
@sjakobi?

Reply:
> Maybe after a few legit PR's between us @sjakobi?

Sounds good to me!